### PR TITLE
MGMT-11566: Default to minimal ISO for tests

### PR DESCRIPTION
**Pull request by CrystalChun**
[MGMT-11566](https://issues.redhat.com/browse/MGMT-11566)
Use the minimal ISO by default to mimic what's
done in production. Also saves time
and space when testing.

/cc @nmagnezi

### CODE REVIEW

**Review by nmagnezi — APPROVED** [review]
/lgtm

File: skipper.env
Comment (CrystalChun) [inline skipper.env:125]: Duplicate https://github.com/openshift/assisted-test-infra/pull/1865/files#diff-473ae8588ab7fd7cf9814e213b7893ed680b19740c25d767f4009791532abb8eL75

File: scripts/deploy_assisted_service.sh
Comment (CrystalChun) [inline scripts/deploy_assisted_service.sh:43]: Needed to include this since infra envs of kube api flow will inherit the assisted service's deployment's value for ISO_IMAGE_TYPE

More details in comment: https://issues.redhat.com/browse/MGMT-11566?focusedCommentId=21019844&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-21019844

File: skipper.env
Comment (nmagnezi) [reply skipper.env:125]: Good catch!

**Review by nmagnezi — APPROVED** [review]
/lgtm

File: scripts/deploy_assisted_service.sh
Comment (eliorerz) [reply scripts/deploy_assisted_service.sh:43]: Is that a bug - `ISO image type can't be specified per infra-env in kube api deployments of assisted-service` ?

File: src/consts/env_defaults.py
Comment (eliorerz) [inline src/consts/env_defaults.py:14]: As a result of this change, all of our test-infra jobs as well as QE e2e tests will be affected.
Need also QE approval - FYI @lalon4 @eliajahshan

File: scripts/deploy_assisted_service.sh
Comment (CrystalChun) [reply scripts/deploy_assisted_service.sh:43]: Not sure if it was intentional - I've asked the slack channel to see if anyone knows! If it's a bug I'll open up a defect and make note to change it in this code once it's fixed.

File: src/consts/env_defaults.py
Comment (CrystalChun) [reply src/consts/env_defaults.py:14]: Thanks for catching that and pinging QE!

File: scripts/deploy_assisted_service.sh
Comment (eliorerz) [reply scripts/deploy_assisted_service.sh:43]: OK, if it's not temporary fix I'm OK with it.
Let's wait for QE approval, but note that it might take a while due to the holiday shutdown.

Comment (CrystalChun) [reply scripts/deploy_assisted_service.sh:43]: Ok sounds good! Not a problem since this is not urgent. Thanks!

File: src/consts/env_defaults.py
Comment (CrystalChun) [reply src/consts/env_defaults.py:14]: Rebumping this @lalon4 @eliajahshan when either of you get a chance, may I have your review

Comment (CrystalChun) [reply src/consts/env_defaults.py:14]: @eliorerz QE has approved!

**Review by osherdp — APPROVED** [review]
looks good to me

**Pull request merged**
